CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ As you'll see in a second, this new version has some important **breaking change
 - **Android** projects are now compiled using the _Android 26 SDK_. The target API level is left unchanged in this release.
   Starting August 2018, new apps submitted to the Play Store will need to target API 26 as a minimum.
   You can now opt your project in to use API 26 (or newer) as the target. Please let us know about any issues, as we'd like to finalize support for Android API 26 by the time `0.57.0` is released.
+  Minimum supported version is **API 14 or Android 4.0**.
```

Comment:
I think that, related to your other comment, you should maybe change this to something like "minimum version set will be 16, but 14 is the lowest number you can set without breaking".

Reply:
RN won't run on android below 16 https://github.com/facebook/react-native/commit/b5dc45420a0d3aa54d2d2075d7f14ff1835df78a.

So I'm closing this PR